src/content/developers/docs/intro-to-ethereum/index.md

```diff
@@ -48,7 +48,7 @@ In practice, participants don't write new code every time they want to request a
 
 At a very basic level, you can think of a smart contract like a sort of vending machine: a script that, when called with certain parameters, performs some actions or computation if certain conditions are satisfied. For example, a simple vendor smart contract could create and assign ownership of a digital asset if the caller sends ETH to a specific recipient.
 
-Any developer can create a smart contract and make it public to the network, using the blockchain as its data layer, for a fee paid to the network. Any user can then call the smart contract to execute its code, again for a fee paid to the network.
+By paying a network fee to use the blockchain as its data layer, any developer can deploy a publicly accessible smart contract with repeatable arbitrary logic. Network participants are then able to call the smart contract code by paying a network fee that incentivises the network to execute the smart contract logic.
```

Comment:
IMO this makes things harder to understand 😕

The first sentence: intention and required action is in wrong order: puts the primary action/object (creating smart contracts) and subject (developers) to the end of the sentence, and the less important information (paying fee) to the beginning. adds more complex language.

The second sentence: uses more complex language that may be harder to understand. adds unnecessary context that may intimidate/confuse users.

Personally, I'd be in favour of not adding these changes.

cc: @konopkja might have an opinion here

Reply:
> 

hm it depends what we want to highlight. Is it the fee necessary for the execution as the new version highlights?  Either way I think both texts can be improved. Namely shorter, simple sentence structures instead of complex sentences.

Reply:
> Is it the fee necessary for the execution as the new version highlights?

@konopkja yes, but the old version also highlights this (IMO, in simpler language). Agree the original could still be improved.